go/protocol/update.go

```diff
@@ -1,5 +1,6 @@
-// Auto-generated by avdl-compiler v1.1.1 (https://github.com/keybase/node-avdl-compiler)
+// Auto-generated by avdl-compiler v1.1.0 (https://github.com/keybase/node-avdl-compiler)
 //   Input file: avdl/update.avdl
+//   Generated : Wed Mar 02 2016 17:02:28 GMT-0800 (PST)
```

Comment:
if you update your node_modules, you'll get a version of the compiler that doesn't timestamp files.